### ISSUE TITLE
Swagger Editor Web doen't recognise Parameter Object when use $ref

### ISSUE THREAD

**Issue by pauloalexandre3d** [swagger-api/swagger-editor#3770]
### Q&A (please complete the following information)
 - OS: Ubuntu 20.04
 - Browser: chrome
 - Version: 109.0.5414.74
 - Method of installation: no installation
 - Swagger-Editor version: https://editor-next.swagger.io/
 - Swagger/OpenAPI version: OpenAPI 3.0.3

### Content & configuration

Example Swagger/OpenAPI definition:
```yaml
openapi: 3.0.3
info:
  title: Swagger Petstore - OpenAPI 3.0
  description: |-
  termsOfService: http://swagger.io/terms/
  contact:
    email: apiteam@swagger.io
  license:
    name: Apache 2.0
    url: http://www.apache.org/licenses/LICENSE-2.0.html
  version: 1.0.11
servers:
  - url: https://petstore3.swagger.io/api/v3

components:
  parameters:
    offsetParam:
      in: query
      name: offset
      required: false
      schema:
        type: integer
        minimum: 0
      description: The number of items to skip before starting to collect the result set.
    limitParam:
      in: query
      name: limit
      required: false
      schema:
        type: integer
        minimum: 1
        maximum: 50
        default: 20
      description: The numbers of items to return.

paths:
  /users:
    get:
      summary: Gets a list of users.
      parameters:
        - $ref: '#/components/parameters/offsetParam'
        - $ref: '#/components/parameters/limitParam'
      responses:
        '200':
          description: OK
  /teams:
    get:
      summary: Gets a list of teams.
      parameters:
        - $ref: '#/components/parameters/offsetParam'
        - $ref: '#/components/parameters/limitParam'
      responses:
        '200':
          description: OK
```

### To reproduce...

1. Copy this yaml (I've got it in here: https://swagger.io/docs/specification/describing-parameters/)
2. Paste it here: https://editor-next.swagger.io/
3. On the bottom, we can see errors:
```shell
Parameter Object must contain one the following fields: content, schema
```

### Screenshots
![image](https://user-images.githubusercontent.com/3245563/214323284-03c42fdd-526b-4bf8-8df6-f10345eee92f.png)

**Comment by char0n** [comment]
Hi @pauloalexandre3d,

Closing as duplicate of https://github.com/swagger-api/swagger-editor/issues/3770
